docs/content/en/command-line-interface/cli.md

```diff
@@ -130,9 +132,19 @@ class GamefaceMyFirstGuic extends BaseComponent {
 
 Note that the `gameface` prefix is added to the component name and to the containing folder.
 
-To build the component run:
 
-1. `npm i` in the component's folder to install the dependencies.
-2. `coherent-guic-cli build` to bundle the source of the component
-3. `coherent-guic-cli build:demo` and load the demo.html folder or
-4. `coherent-guic-cli start:demo` to start a development server that will host the demo page and watch for changes
+# Building the Demo
+
+Each component has `index.html` and `demo.js` files that demonstrate how to use it. You can create a production build of the demo using the `build-demo` command. It will generate the bundle.js and index.html files along with any other assets such as styles located in the **demo** folder. Load the `demo/demo.html` file in a web browser to see the component.

```

Comment:
```suggestion
Each component has `index.html` and `demo.js` files that demonstrate how to use it. You can create a production build of the demo using the `build-demo` command. It will generate the `bundle.js` and `index.html` files along with any other assets such as styles located in the **demo** folder. Load the `demo/demo.html` file in a web browser to see the component.
```

For consistency :D